src/Compilers/Core/Portable/Generated/Operations.xml.Generated.cs

```diff
@@ -5260,6 +5260,55 @@ internal sealed partial class LazyDynamicObjectCreationExpression : BaseDynamicO
         public override IObjectOrCollectionInitializerOperation Initializer => SetParentOperation(_lazyInitializer.Value, this);
     }
 
+    internal abstract partial class BaseDynamicCollectionElementInitializerOperation : HasDynamicArgumentsExpression, IDynamicCollectionElementInitializerOperation
+    {
+        protected BaseDynamicCollectionElementInitializerOperation(SemanticModel semanticModel, SyntaxNode syntax, ITypeSymbol type, Optional<object> constantValue, bool isImplicit) :
+            base(OperationKind.DynamicCollectionElementInitializer, argumentNames: default, argumentRefKinds: default, semanticModel, syntax, type, constantValue, isImplicit)
```

Comment:
>argumentRefKinds: default [](start = 92, length = 25)

I am not sure if default accurately reflects semantics for C#. Should the invocation force non-ref nature of the parameters?